internal/netsync/manager.go

```diff
@@ -37,9 +37,31 @@ const (
 	// peer request queue.
 	maxInFlightBlocks = 16
 
-	// maxRejectedTxns is the maximum number of rejected transactions
-	// hashes to store in memory.
-	maxRejectedTxns = 1000
+	// maxRejectedTxns specifies the maximum number of recently rejected
+	// transactions to track.  This is primarily used to avoid wasting a bunch
+	// of bandwidth from requesting transactions that are already known to be
+	// invalid again from multiple peers, however, it also doubles as DoS
+	// protection against malicious peers.
+	//
+	// It is set to target tracking the maximum number of rejected transactions
+	// that would result from 120 of the 125 default connection slots being all
+	// being filled with malicious peers.  120 is used since it is strictly
+	// greater than the 117 max inbound connections there are 8 outbound
```

Comment:
s / ... 117 max inbound connections there are 8 outbound ...  / ... 117 inbound connections. There are 8 outbound ...

Reply:
Thanks.  Reworded this a bit per the above.